cfme/containers/image.py

```diff
@@ -59,6 +59,44 @@ def perform_smartstate_analysis(self, wait_for_finish=False):
                 raise TimedOutError('Timeout exceeded, Waited too much time for SSA to finish ({}).'
                                     .format(ssa_timeout))
 
+    def check_compliance(self, timeout=240):
```

Comment:
**Required**
Make the wait optional, match `wait_for_finish` attribute in `perform_smartstate_analysis` above for consistency.